src/getFunctions.ts

```diff
@@ -507,6 +510,25 @@ export class GetFunctions {
     }
   }
 
+  getAirQuality(characteristic, _service, _IDs, properties) {
+    const v = parseFloat(properties.value);
+    // 2024 AQI for PM 2.5
+    // https://www.epa.gov/system/files/documents/2024-02/pm-naaqs-air-quality-index-fact-sheet.pdf
+    if (_service.isPM2_5Sensor) {
```

Comment:
Need to check this? Something else also use getAirQuality function?

Reply:
There are air quality sensors to check other things.

<img width="1026" alt="image" src="https://github.com/ilcato/homebridge-fibaro-home-center/assets/52405559/487f61e3-52d0-4549-9c7a-107f9bb3622a">

Reply:
Ok, so this is already a preparation if they were added in the future.

Reply:
> Yes, I want to support other kind of air quality sensors in future.

👍